contrib/vagrant/provision-util.sh

```diff
@@ -81,6 +81,9 @@ os::provision::init-certs() {
   local volumes_dir="/var/lib/openshift.local.volumes"
   local cert_dir="${server_config_dir}/master"
 
+  # Clear stale configuration
+  rm -rf ${config_root}/openshift.local.*
```

Comment:
Does the previous response answer your question, or did you want the explanation provided inline as a comment?